### ISSUE TITLE
Fix toot postion on big screen navigatin with j / k

### ISSUE THREAD

**Issue by Marzal** [cheeaun/phanpy#524]
**Is your feature request related to a problem? Please describe.**
On Desktop next toot changes up and down navigating with the keyboard shortcuts

**Describe the solution you'd like**
Next toot selected is either at the top or at the same vertical height

**Describe alternatives you've considered**
Top seems easy, but the center of the screen would be great.

**Additional context**
https://github.com/cheeaun/phanpy/issues/246#issuecomment-1916259706

Thanks for this amazing client.

**Comment by jan-vandenberg** [comment]
Yes this! The j/k navigation is good, but it needs to **always** display the selected toot at the top. Just like Twitter. 
Because toots are different sizes navigation is yanky and your eyes have to keep scanning where the next toot will end up. If the toot in focus is **always** at the top, that would make for a much better experience!

**Comment by jan-vandenberg** [comment]
I created a PoC client to show what I mean.

Here I j/k scroll through my timeline, and the focussed toot stays fixed at the top. This makes it much easier to scroll the timeline because you don't have to re-adjust / scan where the toot ended up. 

https://piks.nl/upload/upload/explorer_2025-01-27_12-55-29.mp4
